lib/travis/listener.rb

```diff
@@ -15,7 +15,7 @@ def config
 
   module Listener
     class Config < Travis::Config
-      define  redis:   { url: 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
+      define  redis:   { url: ENV['LISTENER_REDIS_URL'] || 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
```

Comment:
With an addon allocated to the listener app (which is my preference, which iiuc is not currently the case), I would expect `REDIS_URL` to be the default.  Is there a reason why we need the env var within the listener app to be `LISTENER_REDIS_URL`?